docs-web/releases.md

```diff
@@ -1,5 +1,9 @@
 # Releases
 
+### 1.11.0
+
+A list of changes can be found on Github at: [1.11 Release](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.11.0)
```

Comment:
Once we finalize the changelog, we'll need to copy it here.  the release date is needed as well.

we publish changelogs for all products at https://docs.nginx.com/